test/test-readability.js

```diff
@@ -201,6 +201,12 @@ describe("Readability API", function() {
       expect(new Readability(doc)._maxElemsToParse).eql(0);
       expect(new Readability(doc, {maxElemsToParse: 42})._maxElemsToParse).eql(42);
     });
+
+    it("should accept a keepClasses option", function() {
```

Comment:
Can you also add a test that checks that the option actually works?

Reply:
Sure, done! 👍